gems/pending/spec/support/test_env_helper.rb

```diff
@@ -0,0 +1,177 @@
+require 'yaml'
+require 'fileutils'
+require 'pathname'
+require 'vcr'
+
+GEMS_PENDING_ROOT ||= File.expand_path(File.join(__dir__, "../.."))
+
+class TestEnvHelper
+  attr_reader :relative_dir, :absolute_recording_dir, :config_file
+
+  SPEC_DIR       = File.join(GEMS_PENDING_ROOT, 'spec')
+  SPEC_PATH      = Pathname.new(SPEC_DIR)
+  RECORDINGS_DIR = File.join(SPEC_DIR, 'recordings')
+
+  # Set TEST_ENV_DIR environment variable, when recording.
+  TEST_ENV_DIR       = ENV['TEST_ENV_DIR'] || File.join(SPEC_DIR, 'test_env')
+  CONFIG_DIR         = File.join(TEST_ENV_DIR, 'config')
+  GLOBAL_CONFIG_FILE = File.join(TEST_ENV_DIR, 'config.yml')
```

Comment:
As with above, prefer Pathnames over `File.join`. More flexible and each thing underneath is just `THING.join()`instead of calling in `File.join(THING, "blah")`